### PR TITLE
[12.0][ADD] account_menu_invoice_refund

### PR DESCRIPTION
**Pull request by kittiu**
By Odoo standard, Invoices and Refunds are in different menu, and can't be mixed upon payment.

This module add 2 new menus,

1. Customers > Invoices / Credit Notes
2. Vendors > Bills / Refunds

Additionally it allow register net payment by selecting both invoice and refund.

**Note:**
This is accomplished by simply remove the condition that disallow mixing invoice and refund in account.payment's default_get()

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Good idea

**Comment by etobella** [comment]
ping @jarroyomorales

**Comment by jarroyomorales** [comment]
LGTM!

File: account_menu_invoice_refund/hooks.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+MAP_INVOICE_TYPE_PARTNER_TYPE = {
+    'out_invoice': 'customer',
+    'out_refund': 'customer',
+    'in_invoice': 'supplier',
+    'in_refund': 'supplier',
+}
+
+
+def post_load_hook():
```
Comment (AaronHForgeFlow) [inline account_menu_invoice_refund/hooks.py:16]: It works so this seems correct, but why not putting it on the account_abstract_payment class?

Comment (kittiu) [reply account_menu_invoice_refund/hooks.py:16]: You mean, why not inherit account.abstract.payment?
I want to remove the checking, I need use hook. I don't know how to do it any other way though.

**Comment by kittiu** [comment]
Ready for merge

File: account_menu_invoice_refund/hooks.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply account_menu_invoice_refund/hooks.py:16]: :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests

**Review by jarroyomorales — APPROVED** [review]

**Pull request closed without merging**